js/cart/cart.js

```diff
@@ -0,0 +1,67 @@
+import { $ } from '../base/base.js';
+import { PRODUCT_STORAGE_KEY } from '../base/constant.js'
+import dialog from '../component/confirm.js'
+
+const tableCart = $(".table-card__tbody")
+const btnDelete = $(".delete-product")
+
+const shoppingCart = {
+    products: JSON.parse(localStorage.getItem(PRODUCT_STORAGE_KEY)) || [],
+
+    loadProduct: function() {
+        const htmls = this.products.map((product, index) => {
```

Comment:
```suggestion
        const htmls = this.products.map((product) => {
```
Vài chỗ đâu thấy dùng tới index đâu, thì gọi làm gì nhỉ